katana/wui/core/templates/core/index.html

```diff
@@ -35,7 +35,7 @@ <h1>
 						<div class="hidden-menu">
 							<ul>
 								<li katana-click="katana.openProfile" url="/katana/settings/" jsurls="/static/settings/js/settings.js,">Profile Settings <i class="fa fa-user"></i></li>
-								<li katana-click="katana.aboutUs">About <i class="fa fa-info"></i></li>
+								<li katana-click="katana.aboutUs" url = "/katana/settings/" jsurls = "/static/settings/js/settings.js">About <i class="fa fa-info"></i></li>
```

Comment:
Hi @hamdih,

There doesn't seem to be a reason why the ```url``` and ```jsUrls``` attributes were included in the new ```<li>``` tag. They can be safely removed.